test/end-to-end.js

```diff
@@ -146,8 +146,8 @@ describe('end-to-end', function() {
         .expect(404, done);
     });
 
-    it('comes with loopback-component-explorer', function(done) {
-      request(app).get('/explorer/swagger.json')
+    it.skip('comes with loopback-component-explorer', function(done) {
```

Comment:
Please add a `TODO` comment pointing to the follow-up issue, one comment per each `it.skip` please.